src/html/html-element-reference-scanner.ts

```diff
@@ -58,7 +58,18 @@ export class HtmlElementReferenceScanner implements HtmlScanner {
 
         elements.push(element);
       }
-    });
+
+      // Descend into templates.
+      if (node.tagName === 'template' && node['content']) {
```

Comment:
I'm debating with myself right now on whether or not `visit` should be visiting template contents automatically. From an "it's an AST" perspective, it's odd that it doesn't. From a "it's DOM, and traverses with .childNodes" perspective, it's probably ok that it doesn't.